db/migrate/20180612074749_add_area_of_work_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAreaOfWorkToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :areaOfWork, :string
```

Comment:
we should use Rails conventions here and call this field: `area_of_work` using snake_case instead of camelCase